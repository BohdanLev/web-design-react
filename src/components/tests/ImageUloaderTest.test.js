import React from 'react'; 
import ImageUploader from '../ImageUploader';
import { shallow } from 'enzyme';


describe('ImageUploaderComponent', () => {
    
      describe('ImageUploader container initial', () => {
        const ImageUploaderContainer = (<ImageUploader/>)

        it('renders properly', () => {
            expect(ImageUploaderContainer).toMatchSnapshot()
          })    
        
          const component = shallow(<ImageUploader />)
          it('ImgUrl default value ', () => {
            expect(component.state().image).toEqual(null)
          })
          
          

     })

     

})