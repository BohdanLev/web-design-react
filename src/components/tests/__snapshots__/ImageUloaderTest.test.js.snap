// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent Footer container initial renders properly 1`] = `<ImageUploader />`;

exports[`ImageUploaderComponent ImageUploader container initial renders properly 1`] = `<ImageUploader />`;
